### PR TITLE
sandboxed-containers-operator: add check job

### PR DESCRIPTION
**Pull request by wainersm**
Added the 'check' job that will run the `make test` in the repository for pull requests.

### CODE REVIEW

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Review by bpradipt — APPROVED** [review]
/lgtm

File: ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-main.yaml
Comment (cpmeadors) [inline ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-main.yaml:6]: OCP 1.4.0 will be build against rhel9, golang 1.19 and OCP 4.13.  Prow should be consistent with our builds.

**Review by cpmeadors — CHANGES_REQUESTED** [review]
The check itself looks good.  Just a comment about the containers used.

**Comment by wainersm** [comment]
/pj-rehearse pull-ci-openshift-sandboxed-containers-operator-main-check

**Comment by wainersm** [comment]
Updated the PR to build the against rhel9, golang 1.19 and OCP 4.13.

The removal (or fixing) of the e2e test will be done in another PR.

**Comment by wainersm** [comment]
@cpmeadors `ci/rehearse/openshift/sandboxed-containers-operator/main/check ` has passed with the new build root. But I still don't understand why tide did not merge it automatically

**Comment by wainersm** [comment]
Apparently it needs at least one lgtm label.

**Review by cpmeadors — APPROVED** [review]
/lgtm

**Pull request merged**
